react/features/base/react/components/web/JitsiSectionListItem.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+
+import React, { Component } from 'react';
+import Button from '@atlaskit/button';
+
+type Props = {
+
+    /**
+     * Room to be rendered in the following format
+     *
+     * {
+     *     colorBase: string,       <- the color base of the avatar
+     *     title: string,           <- item title
+     *     url: string,             <- item url
+     *     lines: Array<string>     <- additional lines to be rendered
+     * }
+     */
+    room: Object,
```

Comment:
This is really not obvious. Give it another name please. We use `room` all across the code and it's always a string. This breaks the "principle of least astonishment".

Reply:
In hindsight `room` definitely wasnt the best name; however, this component has been removed now.